### ISSUE TITLE
where is the little x to clear the input field ?

### ISSUE THREAD

**Issue by GDM70** [running-coder/jquery-typeahead#161]
When I try one of the demonstration examples in my project everything works perfectly but when I combine it with the latest bootstrap version then the little x to clear the input field is no longer visible.

this is my index file 

<!DOCTYPE html>
<html ng-app="myApp">

    <head>

        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="description" content="">
        <meta name="author" content="">
        
        <link rel="icon" href="img/favicon.ico">

        <title>My Project</title>

        <link type="text/css" rel="stylesheet" href="css/bootstrap.css">
        <link type="text/css" rel="stylesheet" href="css/jquery.typeahead.css">
        <link type="text/css" rel="stylesheet" href="css/theme.css">
        
    </head>

    <body>
         
          ....
        
        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.0.7/angular.min.js"></script>
        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.0.7/angular-resource.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.18/angular-ui-router.min.js"></script>

        <script src="http://code.jquery.com/jquery-2.1.0.min.js"></script>
        
        <script src="js/bootstrap/bootstrap.js"></script>
        <script src="js/bootstrap/jquery.typeahead.js"></script>
   
    </body>

</html>

when i remove the line 

<link type="text/css" rel="stylesheet" href="css/bootstrap.css">

everything works fine

I use bootstrap 3.3.6 + JQuery Typeahead 2.3.4

Is this a known issue ?

**Comment by running-coder** [comment]
hey,

it is from the bootstrap styles if I remember correctly, try removing

``` css
input[type="search"] {
  -webkit-box-sizing: content-box;
     -moz-box-sizing: content-box;
          box-sizing: content-box;
  -webkit-appearance: textfield;
}
```

**Comment by running-coder** [comment]
Also refers to #128

**Comment by running-coder** [comment]
Added
``` css
.typeahead-field input[type="search"]::-webkit-search-cancel-button {
  /*Bootstrap 3 override*/
  -webkit-appearance: searchfield-cancel-button;
}
```

Works in Chrome and IE, not Firefox

**Comment by GDM70** [comment]
Thanks ! Now it works.

**Comment by episode17** [comment]
¯\_(ツ)_/¯

**Comment by running-coder** [comment]
you might want to have a look at the latest `2.5.0` version, a default cancel button has been added #174

**Comment by alemol** [comment]
I tried to run a minimal example with tags but the little x just does not appears. I followed all the recommendations and does not work. Does some one could post a full working snippet please?

```
<!DOCTYPE html>
<html lang="en">
<head>

  <meta charset="utf-8">

  <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
  <script src="http://twitter.github.io/typeahead.js/releases/latest/typeahead.bundle.js"></script>
  <link   rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/2.3.2/css/bootstrap.min.css">
  <script src="https://cdn.jsdelivr.net/bootstrap/2.3.2/js/bootstrap.min.js" rel="stylesheet"></script>
  <script src="https://cdn.jsdelivr.net/bootstrap.tagsinput/0.8.0/bootstrap-tagsinput.min.js"></script>

  <style type="text/css">
    .typeahead-field input[type="search"]::-webkit-search-cancel-button {
      /*Bootstrap 3 override*/
      -webkit-appearance: searchfield-cancel-button;
    }
  </style>

</head>

<body>

<select multiple data-role="tagsinput">
  <option value="Amsterdam">Amsterdam</option>
  <option value="Washington">Washington</option>
  <option value="Sydney">Sydney</option>
  <option value="Beijing">Beijing</option>
  <option value="Cairo">Cairo</option>
</select>

</body>
</html>
```
Gives no errors but not the expected output Why??

![screenshot from 2018-07-05 13-30-43](https://user-images.githubusercontent.com/7301106/42341164-c177dcbe-8057-11e8-8660-34a53644f369.png)

**Comment by tombertrand** [comment]
wrong project? this is not bootstrap typeahead

**Comment by davidenco** [comment]
This change is not included in v2.10.6?
